app.json

```diff
@@ -47,6 +47,7 @@
       }
     },
     "android": {
+      "newArchEnabled": false,
```

Comment:
oha ;) https://x.com/tomekzaw_/status/1886741028960424101
do you think this is a proper solution? this means everything in android is on old architecture, not only the map.
did you do some more research and can point to some links?

Reply:
i found this https://github.com/react-native-maps/react-native-maps/issues/5247, which seems identical. there is a comment pointing t some PR with a fix in an android java file

Reply:
Yes. That wouldn't be my first choice either. But I don't see any other solution at this point.
I know this is not a fix. 🤷‍♂️
https://github.com/tomekvenits/react-native-map-clustering/issues/279

Reply:
Yes. It might be a clue for me. I'll look into it closely. Thank you.